modules/ROOT/pages/to-find-info.adoc

```diff
@@ -31,7 +31,7 @@ To select the ranking of search results in the new Exchange search experience, u
 
 In the original Exchange search experience, searches apply only to the Exchange organization you are viewing. If you are viewing a private Exchange organization, the search lists only that organization’s assets and not MuleSoft public assets.
 
-In the new Exchange search experience, you can select which of the organizations visible to you to include in each search. By default, no organization is selected in the menu, and Exchange searches MuleSoft public assets and the assets of the root organization and its business groups.
+In the new Exchange search experience, you can select which of the organizations visible to you to include in each search. 
```

Comment:
Is it possible that an "are" is missing? I find this sentence hard to understand. Should it read?:
"In the new Exchange search experience, you can select which of the organizations _**are**_ visible to you to include in each search". or
In the new Exchange search experience, you can select which of the organizations visible to you **_are_** to **_be included_** in each search.